mewarpx/mewarpx/diags_store/particle_histogram_diagnostic.py

```diff
@@ -145,37 +109,126 @@ def get_fileprefix(self, species):
             f"{self.assembly.name}_{species}_{mwxrun.get_it():010d}"
         )
```

Comment:
I would use `self.name` rather than `self.assembly.name`, which would both allow some histograms not associated with assemblies in the future, and allow two histograms using the same conductor but with different quantities processed - for now ZPlane could set `self.name = self.assembly.name`, and later we could do `self.name = f'positions_{self.assembly.name}` or whatnot.
Make sure to document as a required subclass var, too.